template-contributing-guide.md

```diff
@@ -0,0 +1,341 @@
+  - [Guidelines for choosing a template](#guidelines-for-choosing-a-template)
+- [Research and draft the template](#research-and-draft-the-template)
+  - [Recommended research strategies](#recommended-research-strategies)
+- [Workshop the draft of the template](#workshop-the-draft-of-the-template)
+  - [Accepting feedback from others](#accepting-feedback-from-others)
+- [Submit a pull request](#submit-a-pull-request)
+- [Improve the template with user feedback](#improve-the-template-with-user-feedback)
+
+## Before you start
+
+Before starting, ensure you are familiar with The Good Docs Project’s goals, key concepts, and supported personas. For more information, see [About the Good Docs Project](https://thegooddocsproject.dev/about/).
+
+
+### Template roles and resources
+
+As you work on contributing templates to our project, various resources and members of our community are available to help you along the process. These include:
+
+- **Templateers** - Any individual who contributes to the Good Docs Project (including you!).
+- **The templates coordinator** - This templateer oversees our overall template development process as a project manager and provides assistance to contributors working on templates.
+- **Template mentors** - This group of experienced templateers lead template writing working groups where you can receive guidance and mentorship while working on your template.
+- **Community reviewers** - This group of templateers are available to review templates during the workshop phase. They include members of your template workshop but also members from the larger Good Docs Project community who have volunteered to help review templates.
+- **Pull request reviewers** - This group of experienced templateers review and approve pull requests submitted to the templates repository.
+
+In addition to these roles, these resources are available to help you as you contribute templates to our project:
+
+- **Working groups** - Our project is organized into several working groups that meet on a regular basis to work on the project’s key initiatives. To effectively contribute to a project, consider joining one of these working groups.
+- **Templates issue list** - All the templates that are actively being worked on or which could be worked on have a corresponding issue in the templates repository. Use the issue list or the kanban board to find a template to work on and track your template’s progress as it moves throughout the template writing phases.
+
+These roles and resources (including the names of the individuals currently serving in these roles) are explained in the [Template Roles and Resources](template-roles-and-resources.md) guide.
+
+
+### Required deliverables
+
+Currently, for a template set to be considered complete, each template set should have these documentation deliverables:
+
+<table>
+  <tr>
+    <th>Deliverable</th>
+    <th>Description and purpose</th>
+    <th>Required?</th>
+  </tr>
+  <tr>
+    <td>Raw template</td>

```

Comment:
Raw template does not seem like the right term for this file in the template set. Why not just call it "Template" or "Template File"? Raw implies there is some sort of refinement that has not happened yet. But this file will still be included in the template set after it has undergone all community feedback and revisions. In what sense is it "raw" at that point?

Reply:
Sure, that was just the wording used in the base template, but I'm fine with just "template" or "template file."

Reply:
I've adopted this suggestion. Resolving the comment.